en/docs/consuming-services/create-a-subscription.md

```diff
@@ -1,20 +1,32 @@
+# Subscribe to an API
```

Comment:
This does not require a title because it is reusable content used in both **Manage Subscriptions** and **Consume a Service** pages. The content is also structured to allow reusability.